config/webhook/manifests-new.yaml

```diff
@@ -111,3 +111,24 @@ webhooks:
             - secrets
     sideEffects: None
     admissionReviewVersions: ["v1", "v1beta1"]
+  - clientConfig:
+      caBundle: Cg==
+      service:
+        name: nephe-controller-webhook-service
+        namespace: system
+        path: /validate-v1-configmap
+    failurePolicy: Fail
+    name: vsecret.kb.io
```

Comment:
This is not correct. it needs to be changed to configmap.

Reply:
Solved